css/main.css

```diff
@@ -0,0 +1,100 @@
+/* Define standard variables and values for website */
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 70vh;
+  background-color: #ff4dff;
+}
+
+.page-header {
+  border: none;
+  margin-top: 25%;
+  size-adjust: 190%;
+  -webkit-text-size-adjust: 100%;
+     -moz-text-size-adjust: 100%;
+          text-size-adjust: 100%;
+  font-size-adjust: center;
+  font-weight: bolder;
+  block-size: 100%;
+}
+
+.first-header {
+  padding-top: 1%;
+  border: none;
+  width: 480px;
+  padding-bottom: 40px;
+}
+.first-header h1 {
+  font-family: "Georgia";
+  font-style: normal;
+  font-size: 23px;
+  letter-spacing: normal;
+  width: 480px;
+}
+
+section.input {
+  height: 50px;
+  width: 440px;
+}
+section.input #addName {
+  -webkit-text-size-adjust: 4%;
+     -moz-text-size-adjust: 4%;
+          text-size-adjust: 4%;
+}
+section.input #addNameToList {
+  -webkit-text-decoration: solid;
+          text-decoration: solid;
+  text-decoration-color: rgb(235, 247, 255);
+  -webkit-text-size-adjust: 4%;
+     -moz-text-size-adjust: 4%;
+          text-size-adjust: 4%;
+  text-transform: uppercase;
+}
+section.input input[type=text] {
+  width: 83%;
+  height: 7px;
+  background: rgba(236, 227, 227, 0.8784313725);
+  padding: 5px;
+  font: 100 16px/1.5 "Roboto", sans-serif;
+  transition: border 0.3s linear;
+  vertical-align: middle;
+  line-height: 18px;
+  height: 20px;
+  width: 280px;
+}
+section.input button {
+  vertical-align: middle;
+  line-height: 8px;
+  padding: 6px;
+  border: none;
+  border-radius: 8px;
+  cursor: pointer;
```

Comment:
Using `cursor: pointer` on a button is probably not necessary as that is usually applied to buttons by default

Reply:
This will be removed.